packages/rlc-codegen/src/buildPollingHelper.ts

```diff
@@ -1,21 +1,15 @@
 import { RLCModel } from "./interfaces.js";
-import { readFileSync } from "fs";
 import * as path from "path";
-import * as hbs from "handlebars";
+// @ts-ignore: to fix the handlebars issue
+import hbs from "handlebars";
```

Comment:
This is workaround for module `handlebars`. 

- If we import as below we can't get nothing in runtime in esm.

```typescript
import * as hbs from "handlebars";
```

- If we import the current way it will introduce the TypeError from compiler so I add `@ts-ignore` to ignore this compiler complain.

Reply:
I see. Thanks